src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -336,8 +336,8 @@ object ProtocPlugin extends AutoPlugin {
           name.endsWith(".java") || name.endsWith(".scala")
         })
         .taskValue,
-      unmanagedResourceDirectories += sourceDirectory.value / "protobuf",
-      unmanagedSourceDirectories += sourceDirectory.value / "protobuf"
+      unmanagedResourceDirectories ++= PB.protoSources.value,
```

Comment:
Wouldn't this cause `package` to pack external `protobuf-src` dependencies coming through PB.protoSources? If not, can it be confirmed with a test?

Reply:
Indeed. I have updated the implementation, clarified the purpose of this PR and added tests.